backend/testfiles/httphandler/_simple-request-headers.test

```diff
@@ -1,9 +1,9 @@
 [http-handler GET /]
 (let body =
-   request.headers
+  (request.headers
    |> PACKAGE.Darklang.Stdlib.List.map (fun (k,v) -> "(\"" ++ k ++ "\", \"" ++ v ++ "\")")
```

Comment:
TODO: Bring this test back (having issues parsing `(\""`)

Reply:
maybe inline this TODO/CLEANUP in the file somewhere?